lib/kubeclient/common.rb

```diff
@@ -427,6 +430,18 @@ def api
 
     private
 
+    # Format ditetime according to RFC3339
+    def format_datetime(value)
+      case value
+      when DateTime, Time
+        value.rfc3339(9)
```

Comment:
are you sure this works with Time?
```
[2] pry(main)> Time.now.rfc3339(9)
NoMethodError: undefined method `rfc3339' for 2018-04-30 21:03:57 +0300:Time
[3] pry(main)> DateTime.now.rfc3339(9)
=> "2018-04-30T21:04:20.133891062+03:00"
```

Reply:
Hmm, I checked it in Rails console

```
2.4.3 :116 >   Time.now.rfc3339(9)
 => "2018-04-30T21:28:46.315155000+03:00"
```

but... yeah, in pure Ruby it doesn't work

Reply:
Done